test/models/work_test.rb

```diff
@@ -0,0 +1,78 @@
+require "test_helper"
+
+describe Work do
+  describe 'validations' do
+    before do
+      # Arrange
+      @work = works(:movie)
+    end
+
+    it 'is valid when all fields are present' do
+      # Act
+      result = @work.valid?
+
+      # Assert
+      expect(result).must_equal true
+    end
+
+    it 'is not valid when fields are not present' do
+      @work = Work.new()
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+    end
+  end
+  describe 'spotlight' do
+    it 'spotlight work is instance of work' do
+      @work = works(:movie)
+
+      @works = Work.all
+
+      result = @works.spotlight
+
+      expect(result).must_be_instance_of Work
```

Comment:
Just a note, if the variable is only used inside the `it` block.  You don't need to make it an instance variable

```suggestion
      work = works(:movie)

      works = Work.all

      result = Work.spotlight

      expect(result).must_be_instance_of Work
```